### ISSUE TITLE
`climate.set_preset_mode` with `preset_mode: none` sets target temp to 35C

### ISSUE THREAD

**Issue by phdelodder** [zxdavb/ramses_cc#54]
Executing the `climate.set_preset_mode` service call sets the target temperature to 35 degrees. 

```yaml
service: climate.set_preset_mode
data:
  preset_mode: None
  entity_id: climate.speelkamer
```

This is on release **0.22.3**. I don’t know when it started to happen, just notice it.

**Comment by zxdavb** [comment]
Thanks @phdelodder for your submission.

Can you confirm the release of **ramses_cc** that you are using.

Can you confirm that you're trying to set the zone to **Follow Schedule** mode and that the following equivalent service call works as expected:
```yaml
service: ramses_cc.reset_zone_mode
data:
  entity_id: climate.speelkamer
```

For other reading this thread - I recommend you always use the native service calls (`ramses_cc.xxxxxx`) where possible.

**Comment by phdelodder** [comment]
Version: 0.22.3

**Comment by phdelodder** [comment]
`ramses_cc.reset_zone_mode` works as intended

**Comment by zxdavb** [comment]
I am using:
```yaml
service: climate.set_preset_mode
data:
  preset_mode: none
target:
  entity_id: climate.01_145038_00
  ```

**Comment by zxdavb** [comment]
@phdelodder I believe this issues is fixed in **0.31.16** / **0.41.16** - is that correct?

**Comment by phdelodder** [comment]
@zxdavb Yes I believe so.
